src/main/java/com/microsoft/azure/vmagent/AzureVMAgentCleanUpTask.java

```diff
@@ -138,8 +147,8 @@ protected DeploymentRegistrar() {
                 ois = new ObjectInputStream(new FileInputStream(OUTPUT_FILE));
                 deploymentsToClean = (ConcurrentLinkedQueue<DeploymentInfo>) ois.readObject();
             } catch (Exception e) {
-                LOGGER.log(Level.WARNING,
-                        "AzureVMAgentCleanUpTask: readResolve: Cannot deserialize deploymentsToClean", e);
+                LOGGER.log(Level.INFO,
```

Comment:
Better to check the existence of the file and only ignore the warning in that case.